typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -124,6 +124,7 @@ export class EvmIsmModule extends HyperlaneModule<
 
     this.chain = this.multiProvider.getChainName(this.args.chain);
     this.domainId = this.multiProvider.getDomainId(this.chain);
+    this.chainId = this.multiProvider.getEvmChainId(this.chain);
```

Comment:
I think the `domainId` field can be removed because I don't see it being used anymore in this class.

Reply:
good spot